tests/integration/find-test.js

```diff
@@ -59,3 +59,10 @@ test('find helper can use (optional) element as the context to query', function(
   actual = find('option[selected]', actual);
   assert.strictEqual(actual, expected, 'option found within select element');
 });
+
+test('find can use window and document', function(assert) {
+  this.render(hbs`<body></body>`);
```

Comment:
I believe that having two bodies is probably ilegal HTML, so I'd rather make this render a div or some text, because the document and the body are going to be present whatever you render.